### DISCUSSION TITLE
Failed to get remote logs, the zip archive is incomplete.

### DISCUSSION THREAD

**Discussion by RodrigoVila** [community/community#116891] [Codespaces]
### Select Topic Area

Bug

### Body

Good evening! I'm trying to get Codespaces logs in order to debug an issue we are having with my company when we try to access to this product.

I'm following this troubleshooting site https://docs.github.com/en/codespaces/troubleshooting/github-codespaces-logs#codespace-logs-1 but when I try to download the "logs.zip" I get an error 

"Failed to get remote logs, the zip archive is incomplete"

Even if I get the message "zip created successfully", the zip is not complete.

A gif is worth a thousand words
![codespaces](https://github.com/community/community/assets/42290738/4d9ff125-d17b-4cf4-af02-17e998889cba)

Thanks in advance

**Comment by arthurauffray** [comment]
Hey! I definitely agree with your statement on gifs haha!

Dumb suggestion, but have you tried opening the codespace in your browser? I would also recommend saving everything in the codespace, "putting it to sleep", closing codespace tabs and VSC, then re-opening them a few minutes later and trying to export logs.

Hope this helps.
